### PR TITLE
Python3 Compatible

### PR DESCRIPTION
**Pull request by ahmadpak**
Python 3 Compatible.

### CODE REVIEW

File: example.py [hunk 1]
```diff
@@ -42,10 +42,10 @@ def sync():
 
 			if name:
 				client.update(doc)
-				print "Updated " + row[EMAIL]
+				print(("Updated " + row[EMAIL]))
```
Comment (Thunderbottom) [inline example.py:45]: there's no need for double brackets in here and in other places

File: frappeclient/frappeclient.py [hunk 2]
```diff
@@ -1,11 +1,11 @@
 import requests
 import json
-
+from urllib.parse import quote
```
Comment (Thunderbottom) [inline frappeclient/frappeclient.py:3]: if we are doing this, we might as well only allow frappe client to be setup on python 3

Comment (ahmadpak) [reply frappeclient/frappeclient.py:3]: How about adding this block
`try:`
`	from urllib.parse import quote`
`except:`
`	from urllib import pathname2url as quote`

Comment (Thunderbottom) [reply frappeclient/frappeclient.py]: there's quote in both python2 and python3

```python
try:
    from urllib import quote
except ImportError:
    from urllib.parse import quote
```

but I am wondering if we should completely drop support for python2 now.

File: frappeclient/frappeclient.py [hunk 3]
```diff
@@ -1,11 +1,14 @@
 import requests
 import json
 
-
+try:
+	from urllib.parse import quote
+except:
+	from urllib import pathname2url as quote
```
Comment (Thunderbottom) [inline frappeclient/frappeclient.py:7]: there is quote in python2 urllib:

```suggestion
	from urllib import quote
```

File: frappeclient/frappeclient.py (hunk 2 above)
Comment (ahmadpak) [reply frappeclient/frappeclient.py]: @Thunderbottom So have you decided which direction we are going ?

Comment (Thunderbottom) [reply frappeclient/frappeclient.py]: @ahmadpak, we can drop python2 support from here

Comment (ahmadpak) [reply frappeclient/frappeclient.py]: Ok great I will do the suggest changes soon.

**Comment by ahmadpak** [comment]
@Thunderbottom Please review

**Pull request merged**